flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/source/NewlyAddedTableITCase.java

```diff
@@ -310,6 +331,277 @@ public void testRemoveTable() throws Exception {
+                            String.format(
+                                    "+I[1016, user_17, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[1013, user_14, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[118, user_7, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[1014, user_15, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[111, user_6, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[2000, user_21, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[109, user_4, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[110, user_5, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[103, user_3, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[101, user_1, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[102, user_2, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[123, user_9, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[1019, user_20, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[121, user_8, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[1017, user_18, Shanghai, 123567891234, %s]",
+                                    tableInThisRound),
+                            String.format(
+                                    "+I[1018, user_19, Shanghai, 123567891234, %s]",
+                                    tableInThisRound));
```

Comment:
If we rename `tableInThisRound` to `changedTable` and use static import from `String.format` to `format`, the code will be more readable.